.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!-- Please see https://github.com/zulip/zulip-terminal#contributor-guidelines ! -->
+
+**What does this PR do?**
+
+Overall description goes here!
+
+- [ ] New Feature?
+- [ ] Fixes Bug? [Fixes <issue>; Partially <issue> | Fixes point <n> of <issue> | Resolves an issue that's not filed]
+- [ ] Internal Refactor?
+
+**Tested?** <!-- Fine to leave some of these unchecked! -->
+- [ ] Manually
+- [ ] Tests adapted (if necessary)
+- [ ] Tests added/changed for any new behavior
```

Comment:
Does the "adapted" and "changed" carry the same meaning here? Then we could remove the first check-box?

Reply:
The intent is to ensure that existing tests still pass, via slight changes if necessary - which is distinct from covering different new behavior. Perhaps the two should be explicitly "Existing tests pass (adapted if necessary)" and "New tests added (if new behavior)". Maybe they're too long though.

Reply:
Maybe we could follow the same wording as in the commit style guide?
Tests added (new feature)
Tests updated (refactor/feature that needs minor amending in existing tests)

Reply:
@Rohitth007 I think that'd require the contributor to explicitly write details in parenthesis? I believe we're keener on the tests in the PR in general, and specific additions/amendments are addressed in commits in any case.

Reply:
@Ezio-Sarthak No, I just used the parenthesis for explaining added vs updated here